pkgs/development/libraries/pupnp/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libupnp";
-  version = "1.12.0";
+  version = "1.14.0";
```

Comment:
This should be removed from this PR (done in #93048).

Reply:
agreed, as mentioned in the other diff, it was just to show it indeed fixes what i claim it fixes